specification/workloads/resource-manager/Microsoft.Workloads/SAPVirtualInstance/readme.md

```diff
@@ -44,6 +44,9 @@ suppressions:
   - code: BodyTopLevelProperties
     from: SAPVirtualInstance.json
     reason: This is an incorrect failure due to a bug in the tool
+  - code: AvoidAnonymousTypes
+    from: SAPVirtualInstance.json
+    reason: This is an incorrect failure due to a bug in the tool
```

Comment:
Please provide a link to the specific issue, so we can trace when the underlying issue is fixed, and the suppression can be removed.  Is it this issue?

https://github.com/Azure/typespec-azure/issues/1163#issuecomment-2462531011

Reply:
If this only applies to one (or a few) places in your spec file, you should also add a `where` clause to limit the scope.

Reply:
Thanks @mikeharder, I have added the link for the issue in the `reason ` and also added `where ` clause to limit the scope of the suppression.